charts/applicationset-progressive-sync/templates/deployment.yaml

```diff
@@ -48,6 +48,8 @@ spec:
             {{- if or (gt ( .Values.replicaCount | int64) 1) .Values.args.enableLeaderElection }}
             - --enable-leader-election=true
             {{- end }}
+            - --namespace={{ .Values.args.Namespace }}
+            - --argo-namespace={{ .Values.args.argoNamespace }}
```

Comment:
perhaps defaulting to the same? (if empty, fill it with the namespace?

Reply:
They already default to the same, but I think I'm missing your suggestion.

Reply:
Ok I'll add some suggestions instead